pkg/apis/gcp/cache/v1alpha1/cloudmemorystore_instance_types.go

```diff
@@ -18,9 +18,8 @@ package v1alpha1
 
 import (
 	"strconv"
-	"strings"
 
-	redis "google.golang.org/genproto/googleapis/cloud/redis/v1"
+	"google.golang.org/genproto/googleapis/cloud/redis/v1"
```

Comment:
does this redis v1 import not conflict with the core v1 import right below it?

Reply:
No it doesn't,  version (v1) is skipped and import resolved to the next value, `redis` in this case. Hence, `redis` alias is redundant. I will update to `corev1` alias for better readability.